srcpkgs/batsignal/patches/cflags.patch

```diff
@@ -0,0 +1,11 @@
+--- batsignal-1.1.2/Makefile	2020-07-29 05:26:25.000000000 +0100
++++ working/Makefile	2020-07-31 18:18:32.325480448 +0100
```

Comment:
Should be just `Makefile` in both lines.

Reply:
`-Np1` will pick it up just fine, I think.